addons/product_expiry/views/stock_quant_views.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding='UTF-8'?>
 <odoo>
     <record id="view_stock_quant_tree_expiry" model="ir.ui.view">
-        <field name="name">stock.quant.inherit.form</field>
+        <field name="name">stock.quant.inherit.tree</field>
```

Comment:
We could start to follow xml guideline https://www.odoo.com/documentation/16.0/fr/contributing/development/coding_guidelines.html#inheriting-xml